bin/vyper

```diff
@@ -9,7 +9,7 @@ from vyper.parser.parser import parse_to_lll
 from vyper.parser import parser_utils
 from vyper import compile_lll
 
-sys.tracebacklimit = 0
+sys.tracebacklimit = 10
```

Comment:
Why this change?

Reply:
slipped through, I think I will make this so you can set it with an env variable.